Modules/_collectionsmodule.c

```diff
@@ -1860,7 +1945,7 @@ dequeiter_next(dequeiterobject *it)
     assert (!(it->b == it->deque->rightblock &&
               it->index > it->deque->rightindex));
 
-    item = it->b->data[it->index];
+    PyObject *item = it->b->data[it->index];
```

Comment:
I think we should move the declaration of `item` back to the beginning of the scope.